packages/shared/src/components/squads/layout/useSquadDirectoryLayout.ts

```diff
@@ -35,7 +35,7 @@ export const useSquadDirectoryLayout = (): SquadDirectoryLayoutReturn => {
     return flatCategories.reduce(
       (result, { node }) => ({
         ...result,
-        [node.title]: `/squads/discover/${node.id}`,
+        [node.title]: `/squads/discover/${node.title.toLowerCase()}`,
```

Comment:
I see now we do this, do we maybe want to use generated column for "slug" on API (like we do for post) so we then always have reliable slug over which we can properly search? Or even just add slug to the API?

Reply:
I think it simplifies the whole searching since actually you don't search, you just want to match by slug which is better for SEO?

Reply:
I agree with Ante, it's a generic approach as we did for posts, we should uniform it like we did with slug for posts (calculated column on API side)
You can then use findBy({id || slug}) it should make it more uniform/easier.

Reply:
Sure, why not.